JetMETCorrections/Type1MET/interface/CorrectedMETProducerT.h

```diff
@@ -82,6 +84,13 @@ class CorrectedMETProducerT : public edm::stream::EDProducer<>
     delete algorithm_;
   }
     
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    desc.add<std::string>("@module_label");
+    desc.add<edm::InputTag>("src");
+    descriptions.addDefault(desc);
```

Comment:
In HLT we use descriptions.add. I do not know what descriptions.addDefault does and whether it will be fine. Look at the generated cfi file and check!